eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/DeprecatedVariantMongoReader.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.release.io;
+
+import org.springframework.batch.item.data.MongoItemReader;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+import uk.ac.ebi.eva.accession.core.persistence.DbsnpClusteredVariantOperationEntity;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DeprecatedVariantMongoReader extends MongoItemReader<DbsnpClusteredVariantOperationEntity> {
```

Comment:
MongoItemReader is paginated. do we want this? in eva-pipeline we have the MongoDbCursorItemReader.

Using that one you would only need an extra read mehod were you use the mongoTemplate.getConverter to convert the DBObject into DbsnpClusteredVariantOperationEntity.

Reply:
The reason we use the cursor one in eva-pipeline is because the pagination will break the annotation process. This should not be the case and I think pagination is fine.